docs/ref/middleware.txt

```diff
@@ -502,26 +502,41 @@ every incoming ``HttpRequest`` object. See :ref:`Authentication in web requests
 
 .. versionadded:: 5.1
 
-Redirects all unauthenticated requests to a login page. For admin views, this
-redirects to the admin login. For all other views, this will redirect to
-:setting:`settings.LOGIN_URL <LOGIN_URL>`. This can be customized by using the
+Redirects all unauthenticated requests to a login page, except for views
+excluded with :func:`~.django.contrib.auth.decorators.login_not_required`. The
+login page defaults to :setting:`settings.LOGIN_URL <LOGIN_URL>`, but can be
+customized.
```

Comment:
Hoisted mention of `@login_not_required`. Dropped the admin reference, because auth is a lower level than admin and we kinda don’t need to document what the admin does anyway, it’s just sensible.